.circleci/config.yml

```diff
@@ -354,22 +349,23 @@ workflows:
               only:
                 - main
           requires:
+            - approval
```

Comment:
`release-package` requires `approval` so adding it here (and below) is redundant, no?

Reply:
Just a little extra safe-guarding in case anyone removes `release-package` as a required job here for whatever purpose (testing, dev deployments, etc...)

Reply:
Oh, nvm, Pete removed these.